src/Configuration.php

```diff
@@ -301,4 +301,6 @@ public function getSecurityConfig()
     {
         return $this->configuration['security'] ?? [];
     }
+
+    public const CONFIG_NAME_APIKEY = "datahub_apikey"; 
```

Comment:
please move this up and prefix it with the "topic". So something like SECURITYCONFIG_AUTH_APIKEY